src/iceberg/manifest_writer.h

```diff
@@ -103,6 +124,10 @@ class ICEBERG_EXPORT ManifestWriter {
   static constexpr int64_t kBatchSize = 1024;
   std::unique_ptr<Writer> writer_;
   std::unique_ptr<ManifestEntryAdapter> adapter_;
+  bool closed_{false};
+  std::string manifest_location_;
+  // TODO(zhjwpku): consider passing key metadata when makeing ManifestWriter.
```

Comment:
Please file an issue and use the tracking id instead of your id. I know other code already used personal id, but it's not helpful to follow what to do.